### PR TITLE
Nazar/feature/settings v3 left panel

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import style from "./BusinessInfo.module.css";
+import { LeftPanelType } from "../../../../../../main.type";
+import { AiTwotoneEdit } from "react-icons/ai";
+import {
+  settingsViewKey,
+  SETTINGS_VIEW_TEXT_DATA,
+} from "../../../../../../constants";
```
Comment (kipod) [inline frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfo.tsx:7]: IMHO using [baseURL](https://www.typescriptlang.org/tsconfig#baseUrl) can simplify import paths

**Review by kipod — COMMENTED** [review]
Thank you

**Review by kipod — APPROVED** [review]

File: frontend/src/services/settingsViewService.ts [hunk 2]
```diff
@@ -38,3 +39,61 @@ export const changePasswordRequest = async (data: {
...
+      method: "POST",
+      headers: headers,
+      body: formData,
+    }
+  );
+
+  if (!res.ok) {
+    console.error("Bad login");
+    console.error("Can't change business logo");
+    return;
```
Comment (Taraslysak) [inline frontend/src/services/settingsViewService.ts:93]: No further error handling?

**Review by Taraslysak — APPROVED** [review]

File: frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfo.tsx (hunk 1 above)
Comment (Nazar-kob) [reply frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfo.tsx:7]: Thanks I will use it in future

File: frontend/src/services/settingsViewService.ts (hunk 2 above)
Comment (Nazar-kob) [reply frontend/src/services/settingsViewService.ts:93]: Yes

**Pull request merged**